### ISSUE TITLE
Mystery + crossed has no key_logic assigned to the world for that player

### ISSUE THREAD

**Issue by Berserker66** [aerinon/ALttPDoorRandomizer#27]
```
Traceback (most recent call last):
  File "E:\ROMS\Randomizer\ALttPDoorRandomizer-DoorDev\Mystery.py", line 223, in <module>
    main()
  File "E:\ROMS\Randomizer\ALttPDoorRandomizer-DoorDev\Mystery.py", line 101, in main
    DRMain(erargs, seed)
  File "E:\ROMS\Randomizer\ALttPDoorRandomizer-DoorDev\Main.py", line 117, in main
    set_rules(world, player)
  File "E:\ROMS\Randomizer\ALttPDoorRandomizer-DoorDev\Rules.py", line 26, in set_rules
    global_rules(world, player)
  File "E:\ROMS\Randomizer\ALttPDoorRandomizer-DoorDev\Rules.py", line 312, in global_rules
    add_key_logic_rules(world, player)
  File "E:\ROMS\Randomizer\ALttPDoorRandomizer-DoorDev\Rules.py", line 1411, in add_key_logic_rules
    key_logic = world.key_logic[player]
KeyError: 2
```

with these files:
https://cdn.discordapp.com/attachments/662745528316198942/670579265062043648/Berserker.yaml
https://cdn.discordapp.com/attachments/662745528316198942/670579271596507136/Frost.yaml

and this command:
`.\Mystery.py --multi 2 --p1 Players\Berserker.yaml --p2 Players\Frost.yaml --names Berserker,Frost --enemizercli EnemizerCLI/EnemizerCLI.Core.exe --outputpath MultiMystery --create_spoiler`

but any invocation of mystery with someone on crossed seems to give this result.

**Comment by Berserker66** [comment]
Turns out the error occurs because in frost.yaml it says cross instead of crossed.
So I guess the request has changed into, please add a test that the option actually exists.

**Comment by compiling** [comment]
Fixed in 0.0.8-pre.  Log an error and throw an exception if the door shuffle option is not valid.
